packages/components/package.json

```diff
@@ -40,6 +40,7 @@
     "@floating-ui/dom": "^1.6.3",
     "@hashicorp/design-system-tokens": "^2.1.0",
     "@hashicorp/ember-flight-icons": "^5.0.1",
+    "@oddbird/popover-polyfill": "^0.4.1",
```

Comment:
note: this was bumped to `0.4.3` (includes a couple of fixes) since we started working on this component; I'll leave it to you if you want to bump it to the latest before getting it in

Reply:
Good idea, bumped the version. 👍